src/python/grpcio_tests/tests/unit/_alts_credentials_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test of RPCs made using ALTS credentials."""
+
+import unittest
+import os
+from concurrent.futures import ThreadPoolExecutor
+import grpc
+
+
+REQUEST = b'abc'
+
+
+class _GenericHandler(grpc.GenericRpcHandler):
+
+    def service(self, handler_call_details):
+        return grpc.unary_unary_rpc_method_handler(
+            lambda request, unused_context: request)
+
+
+class ALTSCredentialsTest(unittest.TestCase):
```

Comment:
Not sure if we want need this E2E unit test here as it will not work for users who download gRPC python on their machines and run it locally. I think it's sufficient to have the interop test below that runs continuously against every submitted PR.

Reply:
Valid concern. Removing unit test given interop one passed.